### ISSUE TITLE
Make the program open source

### ISSUE THREAD

**Issue by Random06457** [Random0666/Wii-U-Title-Boot-Editor#1]

**Comment by Random06457** [comment]
@Roet-Ivar
> Random first of all, I'm so sorry for doing this, but I haven't found any other way to contact you :(
Is there any chance you'd share or open source your CS-project of the[ wii-u-boot-title-editor](https://github.com/Random06457/Wii-U-Title-Boot-Editor)?
I really like that tool and would love to continue adding some features to it.
Again, I'm sorry for taking this route!

Hi, as you can see it's been a while I haven't touched this project (this issue alone dates from almost 5 years ago). However this was a cool project and I wouldn't mind working on it today.
Unfortunately, as of right now, there are a couple of issues with open-sourcing the project:
- I currently don't have access to the source code as it is stored on the hard drive of a computer that I don't use anymore and that's far away from me. However I think I could manage to get it back in maybe a month or so.
- I believe I would probably be a bit uncomfortable sharing the code I wrote 5 years ago as my quality standards have changed so much.

I think the best solution here would be to rewrite the code as a whole which would allow not only to increase code quality but also to make a version that's supported on more platforms (especially Linux which is my daily driver nowadays).

However, in the case where I rewrite everything, another problem that I have is that I don't have a setup to  test this anymore. I'm currently far away from my Wii U and I think the charger of my Wii U is broken. However I believe this could be arranged if someone would do the tests for me as it wouldn't be too difficult to debug (This program is only a fancy ftp client after all). If you're interested in helping out, you can reach me on discord at `random6457`.

Finally, while I may show interest in the moment, please not that I can't promise anything right now and I wouldn't want to get your hops up if nothing ends up happening.

TL;DR
I would actually be interested in working on this again. But I would prefer to rewrite everything (likely in C++). However I can't test anything right now because I don't have a working Wii U, but if you think you could help me out, please reach out to me on discord at `random6457`.

**Comment by Roet-Ivar** [comment]
Discord is kind of weird and says that we don't share any servers and won't send the message. Sent a friend request as Sodastream 👍

**Comment by Random06457** [comment]
I didn't receive anything so you may have entered my id wrong (?)

And when I try to send a friend request to `Sodastream`, it says that you have incoming friend requests from people you don't share any server with disabled, and that you must be the one to add me.

If you want to enable receiving other friends requests, you'd have to update your `Privacy and Safety` settings in discord.

**Comment by Roet-Ivar** [comment]
Edited

**Comment by Random06457** [comment]
I sent a friend request.
